### PR TITLE
Fix: make OperatorInfo receipts table reset values on cancel

### PR DESCRIPTION
**Pull request by hellypi**

### CODE REVIEW

**Comment by hellypi** [comment]
[Boolean tables not updating when clicking 'cancel'](https://trello.com/c/bkH397sE/501-boolean-tables-not-updating-when-clicking-cancel)

**Review by chaotixkilla — APPROVED** [review]

File: new-lamassu-admin/src/components/booleanPropertiesTable/BooleanPropertiesTable.js [hunk 1]
```diff
@@ -26,7 +27,8 @@ const BooleanCell = ({ name }) => {
 
 const BooleanPropertiesTable = memo(
   ({ title, disabled, data, elements, save, forcedEditing = false }) => {
-    const initialValues = _.fromPairs(elements.map(it => [it.name, '']))
+    const defaultValues = _.fromPairs(elements.map(it => [it.name, 'false']))
+    const initialValues = R.mergeRight(defaultValues, data)
```
Comment (RafaelTaranto) [inline new-lamassu-admin/src/components/booleanPropertiesTable/BooleanPropertiesTable.js]: Why the merge right? Might as well default to

**Review by RafaelTaranto — CHANGES_REQUESTED** [review]

File: new-lamassu-admin/src/components/booleanPropertiesTable/BooleanPropertiesTable.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import { makeStyles } from '@material-ui/core/styles'
 import classnames from 'classnames'
 import { useFormikContext, Form, Formik, Field as FormikField } from 'formik'
-import _ from 'lodash'
+import _ from 'lodash/fp'
```
Comment (RafaelTaranto) [inline new-lamassu-admin/src/components/booleanPropertiesTable/BooleanPropertiesTable.js]: Hmm, just noticing that this was using lodash, we don't even have lodash on the front-end build scripts. Replace this with ramda.

File: new-lamassu-admin/src/components/booleanPropertiesTable/BooleanPropertiesTable.js [hunk 3]
```diff
@@ -26,7 +26,9 @@ const BooleanCell = ({ name }) => {
 
 const BooleanPropertiesTable = memo(
   ({ title, disabled, data, elements, save, forcedEditing = false }) => {
-    const initialValues = _.fromPairs(elements.map(it => [it.name, '']))
+    const initialValues = _.fromPairs(
+      elements.map(it => [it.name, data[it.name] || null])
```
Comment (RafaelTaranto) [inline new-lamassu-admin/src/components/booleanPropertiesTable/BooleanPropertiesTable.js]: Might as well use `??` instead of `||` here.

**Review by RafaelTaranto — CHANGES_REQUESTED** [review]

**Review by RafaelTaranto — APPROVED** [review]

**Pull request closed without merging**